boards/DIM/Inc/App/App_ErrorTable.h

```diff
@@ -0,0 +1,87 @@
+#pragma once
+
+#include <stdint.h>
+
+// clang-format off
+enum ErrorId
+{
+    BMS_STACK_WATERMARK_ABOVE_THRESHOLD_TASK1HZ = 0,
+    BMS_CRITICAL_ERROR_DUMMY_CRITICAL_ERROR = 1,
+
+    DCM_STACK_WATERMARK_ABOVE_THRESHOLD_TASK1HZ = 0,
+    DCM_CRITICAL_ERROR_DUMMY_CRITICAL_ERROR = 1,
+
+    FSM_STACK_WATERMARK_ABOVE_THRESHOLD_TASK1HZ = 0,
+    FSM_CRITICAL_ERROR_DUMMY_CRITICAL_ERROR = 1,
+
+    PDM_STACK_WATERMARK_ABOVE_THRESHOLD_TASK1HZ = 0,
+    PDM_CRITICAL_ERROR_DUMMY_CRITICAL_ERROR = 1,
+
+    DIM_STACK_WATERMARK_ABOVE_THRESHOLD_TASK1HZ = 0,
+    DIM_CRITICAL_ERROR_DUMMY_CRITICAL_ERROR = 1,
+
+    NUM_ERRORS = 10, // Number of non-critical and critical errors combined
+                     // across ALL boards
+};
+
+struct Error
+{
+    // CAN message ID
+    uint32_t std_id;
+    // Each error ID is unique in the error table
```

Comment:
Do you mean in the `CAN` message? If the error ID is globally unique (ie. across all CAN messages) then there's no point to having the `std_id` here as well?

Reply:
Good catch. This comment is wrong. The error IDs will be unique only on a per-board basis to fulfill:
![image](https://user-images.githubusercontent.com/16970019/82135690-63112280-97ba-11ea-9bb0-3f9d990a4a59.png)

Reply:
Initially I thought about making the error IDs globally unique, but considering the fact that we only have 2 digits to display errors that would leave us with 99 errors in total across 5 PCBs. Which seemed too restrictive.

Reply:
The per-board unique IDs are reflected in the enum at the top of this file